.github/workflows/prod-deployment.yaml

```
@@ -0,0 +1,82 @@
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Set TAG environment variable
+      run: echo "TAG=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+
+    - name: Debug TAG value
+      run: echo "TAG value - ${{ env.TAG }}"
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
```

actionlint (1.7.7)

the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)